app/controllers/host_controller.rb

```diff
@@ -646,20 +647,18 @@ def get_hosts
   end
 
   def get_session_data
+    super
     @title      = _("Hosts")
```

Comment:
`@title` should live in the `title` method now, which is being used by the mixin so you should no longer have to set it here..